pkg/sql/alter_table_set_schema.go

```diff
@@ -159,14 +125,9 @@ func (n *alterTableSetSchemaNode) startExec(params runParams) error {
 	newTbKey := sqlbase.MakeObjectNameKey(ctx, p.ExecCfg().Settings,
 		databaseID, desiredSchemaID, tableDesc.Name).Key(p.ExecCfg().Codec)
 
-	b := &kv.Batch{}
-	if params.p.extendedEvalCtx.Tracing.KVTracingEnabled() {
-		log.VEventf(ctx, 2, "CPut %s -> %d", newTbKey, tableDesc.ID)
-	}
-
-	b.CPut(newTbKey, tableDesc.ID, nil)
-
-	return p.txn.Run(ctx, b)
+	return p.writeNameKeyToBatch(
```

Comment:
don't include "ToBatch" now that there isn't a batch anywhere

Reply:
Done